oci/pull.bzl

```diff
@@ -225,4 +228,5 @@ def oci_pull(name, image = None, repository = None, registry = None, platforms =
         config = config,
         # TODO(2.0): remove
         config_path = config_path,
+        tags = bazel_tags,
```

Comment:
could you also rename the attribute to be `bazel_tags` instead of just tags ?

Reply:
I'm not following.  The `oci_alias` here is a repository rule, and all rules have the common attribute `tags`.  I wasn't aware that you could rename or alias existing common attributes.  Feel free to modify/make changes to my pull request if there was a way to do that.

Reply:
by default, repository rules only have a single `name` attribute, `tags` is common for build rules. `tags` here in the repository rule context is confusing, it's https://bazel.build/reference/be/common-definitions#typical-attributes 

you can just add a new attribute called `bazel_tags`.

Reply:
ahh - thank you for the clarification.  In the next commit, I've renamed it to `bazel_tags` and added the attribute.

However, I did remove adding the attribute to `oci_alias` because it wasn't being used.  Let me know if you'd still like me to add it to `oci_alias`

Reply:
Resolving this conversation - I see a thumbs up on the last comment and the changes have been approved.